.github/workflows/test-node.js.yml

```diff
@@ -24,5 +24,5 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm install
```

Comment:
We could even shrink this down to one execution: `npm cit` (https://docs.npmjs.com/cli/install-ci-test.html)

Reply:
I just raised it in PR #74 lest it be forgotten:)